standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/common/StatsSetupConst.java

```diff
@@ -173,6 +173,10 @@ public String getAggregator(Configuration conf) {
 
   public static final String FALSE = "false";
 
+  public static final String HIVE_ICEBERG_STATS_SOURCE = "hive.iceberg.stats.source";
```

Comment:
the conf is defined in HiveConf as well, either figure out to use that everywhere if not possible, change that HiveConf definition to use this

```
    HIVE_ICEBERG_STATS_SOURCE(StatsSetupConst.HIVE_ICEBERG_STATS_SOURCE, "iceberg",
        "Use stats from iceberg table snapshot for query planning. This has two values metastore and iceberg"),
```

Reply:
HiveConf exists in hive-common module which is not a dependency of metastore server.
I saw some instances in HiveMetsStoreClient where other constants of HiveConf where hardcoded.
So I assumed that Hive Metastore doesn't want to depend on hive-common.
Probably because HMS is used not only by Hive?
Please let me know if my understanding is incorrect.